### DISCUSSION TITLE
Set Proxies

### DISCUSSION THREAD

**Discussion by mige92** [Yakifo/amqtt#150] [Q&A]
Hi, is there any way to set proxies for mqtt-broker? Tried it with environment variables, but I guess they are not recognized.

**Comment by ajmirsky** [comment]
@mige92 since the broker receives connections (and not make them), the broker doesn't need any particular configuration to receive connections from a proxy. if you're looking for a way for the proxy to authenticate or reject any connection not associated with a proxy, an authentication plugin can be crafted but it depends on which proxy server you're using.

if you could provide a little more detail of what you're trying to do, I'm happy to help and see what can be done to meet your requirements.
